library/CHART-TEMPLATE/values.yaml

```diff
@@ -15,6 +15,15 @@ global:
   nodeSelector: {}
   tolerations: []
   affinity: {}
+  agent:  # This should not touch main/master because it has things that could break all the charts
+    config:
+      global: global
+      global_overriden: test
```

Comment:
As a user, I would expect things that I put in `global` to be applied to all charts (unless overridden). This makes me have two concerns with having `global.agent`:
- Not all charts will have an agent, which might be counterintuitive
- Some charts might have internal options (e.g. env vars) that silently override things I can put here. As a user I don't have visibility on that and might create unexpected behavior.

I wouldn't consider any of this points to be a blocker, but I wonder what are the advantages of having a `global.agent` to offset the downsides.

Reply:
Duplicate of [this](https://github.com/newrelic/helm-charts/pull/757#discussion_r832149573). Resolving.